pdf_to_csv.py

```diff
@@ -32,7 +32,8 @@ def __init__(self):
 
     def generate_csv_file(self, input_pdf_filepath, out_csv_filepath,
                           is_header=True, identify_columns=False,
-                          temp_file_postfix="", check_page_rotation=False):
+                          temp_file_postfix="", check_page_rotation=False,
+                          measure_file=None):
```

Comment:
log_perf or monitor? Rather than measure_file, slightly verbose name would be more helpful?